GRR/script_for_delta_part_2.sql

```diff
@@ -348,7 +360,28 @@ SELECT q_code,
        CAST(d1 AS FLOAT),
        'MG'
 FROM grr_mult
-  JOIN source_data_1 ON fcc = q_code;
+  JOIN source_data_1 ON fcc = q_code
+  and nfc not like 'D%';
+
+INSERT INTO ds_stage
+(
+  DRUG_CONCEPT_CODE,
+  INGREDIENT_CONCEPT_CODE,
+  numerator_value,
+  numerator_unit,
+  denominator_value,
+  denominator_unit
+)
+SELECT q_code,
+       concept_code_1,
+       (TRIM(d1)::FLOAT) / 5 *volume,
```

Comment:
Could you please add comment here? I'm assuming d1/5*volume is for tablespoons, but don't see how you pick only tablespoons.

Reply:
Thank you for response, I added comment for this case.